hw/top_earlgrey/ip/ast/doc/ast_regs.html

```diff
@@ -1,3 +1,66 @@
+<style>
```

Comment:
is this file supposed to be checked in? we do not have html's merged for any other block.